Sources/Overload/OvGame/premake5.lua

```diff
@@ -35,7 +35,8 @@ project "OvGame"
 
 	filter { "configurations:Release" }
 		defines { "NDEBUG" }
-		optimize "On"
+		optimize "Speed"
+		flags { "LinkTimeOptimization" }
```

Comment:
Deprecated in Premake 5.0.0-beta4. Use `linktimeoptimization` API instead.

Reply:
Since we have an older version of premake than the one you're mentionning, the setting "linkoptimization 'value'" isn't available yet.
We'll need to make a bump of premake and modify our premake5.lua files if needed to reflect the change.